scripts/rollup/bundles.js

```diff
@@ -355,7 +355,7 @@ const bundles = [
 
   /******* React DOM Fizz Server External Runtime *******/
   {
-    bundleTypes: [BROWSER_SCRIPT],
+    bundleTypes: [BROWSER_SCRIPT, FB_WWW_DEV, FB_WWW_PROD],
```

Comment:
What's this for? We shouldn't need it

Reply:
I see, I was thinking we would update the internal sync script to copy over the one used by npm. That's how we sync the ESLint plugin, for example. We're trying to trend in the direction of less FB-specific infra in the open source repo.